releasenotes/notes/add-get-snapshot-method-ee4f85a7b06cb373.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Add a method to easily access the classical "snapshots" of the state after taking some
+    measurements. A classical "snapshot" of the state is the dual frame operator associated with the
+    corresponding outcome. The new method returns the snapshot (typically as a product of local
+    operators) associated with the queried outcome (typically a tuple of integers).
```

Comment:
Can you link to the new method in this release note?
Might want to rephrase it slightly to fit it naturally :+1: